frontend/public/components/notification-drawer.tsx

```diff
@@ -132,11 +131,11 @@ export const getAlertActions = (actionsExtensions: ResolvedExtension<AlertAction
 };
 
 const getUpdateNotificationEntries = (
```

Comment:
Interesting name for a utility that only shows upgrade statuses... does our notification drawer not show anything else? 🤔